apps/cli/src/actions/submit/prepare_branches.ts

```diff
@@ -323,3 +325,18 @@ async function getReviewersMaybeInteractively(
 
   return getReviewers(reviewers);
 }
+
+function cliAuthPrecondition(context: TContext): boolean {
```

Comment:
I'd like this to go in `preconditions.ts` but it was causing an error with the test suite that I couldn't figure out so I'm keeping it here (this is the only file it will be used in for now anyways)